src/behaviours/__tests__/utils.tsx

```diff
@@ -33,6 +35,31 @@ export const drag = (element: any) => {
     return controls
 }
 
+export const asyncDrag = (element: any) => {
+    pos.x = 0
+    pos.y = 0
+    fireEvent.mouseDown(element)
+
+    const controls = {
+        to: async (x: number, y: number) => {
+            pos.x = x
+            pos.y = y
+
+            await act(async () => {
+                fireEvent.mouseMove(document.body, { buttons: 1 })
+                await new Promise(resolve => sync.update(resolve))
```

Comment:
kinda leverages internal knowledge about `usePanGesture`, but this is the only way to fix act warnings